manifests/istio-telemetry/prometheusOperator/templates/servicemonitors.yaml

```diff
@@ -116,7 +116,7 @@ spec:
       action: replace
       targetLabel: pod_name
 ---
-{{- if .Values.security.enabled }}
+{{- if .Values.prometheus.provisionPrometheusCert }}
```

Comment:
`prometheusOperator` and prometheus share the same parameter `provisionPrometheusCert`?

Reply:
Yes. Just like how it used to use `security` which was citadel.

this part of the templates is going to go away from another PR anyways though